ts/utils/payment.ts

```diff
@@ -112,8 +112,10 @@ export function walletHasFavoriteAvailablePsp(
 const hasDescriptionPrefix = (description: string) =>
   description.startsWith("/RFA/") ||
   description.startsWith("/RFB/") ||
+  description.startsWith("/RFS/") ||
   description.startsWith("RFA/") ||
-  description.startsWith("RFB/");
+  description.startsWith("RFB/") ||
+  description.startsWith("RFS/");
```

Comment:
Since prefixes may grow, Could this be a refactoring?

```typescript
const prefixes: ReadonlyArray<string> = ["RFA", "RFB", "RFS"];
const hasDescriptionPrefix = (description: string) =>
  prefixes.some(
    p => description.startsWith(`${p}/`) || description.startsWith(`/${p}/`)
  );
```

Reply:
Yes sounds good! Refactoring in cc7d63c